app/src/test/java/nl/tudelft/b_b_w/model/DatabaseHandlerUnitTest.java

```diff
@@ -42,7 +42,7 @@
     private final String iban = "iban";
     private final String publicKey = "publicKey";
     private Block _block;
-    private final int trustValue = 0;
+    private final int trustValue = TrustValues.INITIALIZED.getValue();
```

Comment:
`Could you give me a link and I prefer not a link to a stackoverflow discussion, where multiple people have different reasonings, which are all correct? `
[Source](https://stackoverflow.com/questions/36585250/when-does-the-jit-automatically-inline-methods)
Since this code is not part of any intense loop or executed frequently, I do not think we have to bother with optimization here.